app/assets/javascripts/admin/addon/templates/email-sent.hbs

```diff
@@ -54,16 +54,29 @@
                 "redo"
                 title="admin.email.bounced"
               }}{{/if}}
-            <a href="mailto:{{l.to_address}}">{{l.to_address}}</a>
+            <p><a
+                href="mailto:{{l.to_address}}"
+                title="TO"
+              >{{l.to_address}}</a></p>
+            {{#if l.cc_addresses}}
+              {{#each l.cc_addresses as |cc|}}
+                <p><a href="mailto:{{cc}}" title="CC">{{cc}}</a></p>
+              {{/each}}
+            {{/if}}
```

Comment:
For better UX, could you do a simple "if there is > 5, truncate into a tooltip?". You can consider using floatkit's tooltip.

Reply:
Hmm. I do wonder if it makes sense to put all CC in a tooltip regardless -- will let you decide.